api/client/samples/at-time-query-examples.ipynb

```diff
@@ -0,0 +1,407 @@
+{
```

Comment:
<p>It might be worth pointing out that "forecast" data points usually are available prior to the <code>end_date</code>, to distinguish it from the previous example of LST. Besides, their source_lag might carry no information. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/gro-intelligence/api-client/pull/98/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>

Reply:
Added